static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js

```diff
@@ -1,43 +1,36 @@
 define([
     'common/utils/config',
     'common/modules/ui/sticky',
-    'common/utils/fastdom-promise',
-    'lodash/objects/defaults'
+    'common/utils/fastdom-promise'
 ], function (
     config,
     Sticky,
-    fastdom,
-    defaults
+    fastdom
 ) {
 
-    var mpuHeight = 275,
-        StickyMpu = function ($adSlot, options) {
-            this.$adSlot = $adSlot;
-            this.opts    = defaults(options || {}, {
-                top: 0
-            });
-        };
+    var MPU_HEIGHT = 275;
```

Comment:
Not really I would argue as we use [`const` like variables](https://github.com/guardian/frontend/pull/12007/files#diff-482df1b51d47e064a1c5e155b31ef0ecR25) all over the shop - and this denotes something is different (when I first saw it I thought it was some for of `ENUM`). Just my :dollar:'s worth.